test-network-k8s/README.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Test Network 
+
+This project re-establishes the Hyperledger [test-network](../test-network) as a _production deployment_ on Kubernetes.
+
+_Fabric, Ahoy!_
+
+
+### Objectives:
+
+- Provide **simple**, "one-click" actions for deploying production-grade Fabric networks on Kubernetes.
```

Comment:
being picky could ask what bit is production grade ::)   eg. we've not got load balancing or HA?

Reply:
Understood this is the start of a _slippery slope_.  In the "big vision," all HLF networks will be bootstrapped as "production grade" systems, even local networks running in a disposable Kubernetes cluster.  

That said, I agree this needs a better term that works well with little explanation (or scope creep.)   Where would we draw the line?   What about softHSM?  HA/LB?  What about the state storage?  Or LDAP for managing identity registrations?  Or Intermediate CAs with real SSL root certificates signed by Thawte/Verisign?   Or built-in ledger snapshots that run nightly and sync the blockchain to s3:// ?  Or ....  a `./network` script that doesn't soft-lock the network when re-enrolling existing certificates? 

...